baseplate/observers/sentry.py

```diff
@@ -10,6 +10,11 @@
 
 import raven
 
+from pymemcache.exceptions import MemcacheError
```

Comment:
:no_bell: moot since we need to use strings anyway, but I think we'd want to be more careful about importing this as we don't require pymemcache to be installed for a service unless it's using that integration.